package.json

```diff
@@ -41,6 +41,7 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@types/lodash": "^4.14.159",
+    "apexcharts": "^3.26.2",
```

Comment:
looks like a good, stable library. So it is a native javascript library, hopefully it does not conflict to much with react.